include/swift/AST/ASTContext.h

```diff
@@ -898,6 +898,10 @@ class ASTContext final {
   /// Get the runtime availability of support for concurrency.
   AvailabilityContext getConcurrencyAvailability();
 
+  /// Get the runtime availability of the `DiscardingTaskGroup`,
+  /// and supporting runtime functions function
+  AvailabilityContext getConcurrencyDiscardingTaskGroupAvailability();
```

Comment:
is this related to the fix?

Reply:
No, thank you -- rebase should get rid of this. Cleaning that up now